### DISCUSSION TITLE
Note metadata field

### DISCUSSION THREAD

**Discussion by alexpovel** [jonaswinkler/paperless-ng#1226] [Feature Requests]
Paperless-ng is excellent and its minimalist approach makes it an app that is actually enjoyable to use (and fast!). I'd hate for it go get bloated, its current feature set feels quite complete.

However, there's one thing that would've been useful on multiple occasions now: an additional metadata field for **notes**. I have a bunch of documents that exist in multiple versions, sometimes from different sources (e.g. an empty digital template, a filled out digital version, and a signed, physical scan of that same document). Sometimes even a short note would go a long way of explaining why a document exists, why it was added, what makes it important/different/... 

I created a mock-up:

![image](https://user-images.githubusercontent.com/48824213/128869874-058a9388-db9d-49a8-9362-300591666647.png)

A note would be a free-text field. It is distinct enough from tags and the title field that its existence could be warranted:

- a note can (and probably mostly would be) one-off; one-off tags are somewhat antithetical
- a title should be the actual document title (that's how I do it, principle of least surprise), and so cannot be chosen freely if following that principle.

A note is unstructured and would not be available to sort by etc. It's just down there at the bottom for those occasions where it's needed. 
For example, it could hold synonyms aka an alternative description of the document. On various occasions, I tried to find a document that's associated with term *X* in my mind but was unable to find it. Reason being that the document's full text and title don't actually contain that term *X*, but a closely related one, *Y*. That term *X* could go into the note field, allowing us to find the document through it.

Notes could bring great flexibility. Users might come up with interesting use cases, adding a whole new dimension to paperless-ng. On the other hand (but I don't know this), it might be quite simple to implement, mostly 'only' another line in the code, defining a new metadata field (basically just like the *Title* field but without a character limit and with newline support).

Let me know what you think!

**Comment by frank-from-cologne** [comment]
Yes, a note field would be VERY VERY good.

I would also like it if it could be multi-line.

And if you could search for the strings used here.

That way I would have an additional, simple + very flexible tool.

(I also find CUSTOM METADATA [#274](https://github.com/jonaswinkler/paperless-ng/issues/274)  interesting, but a note field I see independent and additional, right?)

**Reply by frank-from-cologne** [reply]
check also [751](https://github.com/jonaswinkler/paperless-ng/discussions/751)

**Comment by alexpovel** [comment]
This basically became a thing in https://github.com/paperless-ngx/paperless-ngx/pull/1375 , released in [Paperless-ngx v1.9.0](https://github.com/paperless-ngx/paperless-ngx/releases/tag/v1.9.0).
